sdet/deploy/action.yaml

```diff
@@ -23,28 +23,26 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: get allure history
+    - name: get report history
       uses: actions/checkout@v4
-      if: "${{ inputs.platform == 'api' }}"
       continue-on-error: true
       with:
         ref: gh-pages
         path: gh-pages
 
-    - name: mochawesome report action
-      uses: firmanrosid/mochawesome-report-action@v1.0.0
-      if: "${{ inputs.platform == 'api' }}"
```

Comment:
ini juga

Reply:
updated

Reply:
kalau lu delete parameter **platform** and **custom_command_report_folder** makesure inputs param criteria yang dibagian atas juga di delete ya @firmanrosid

Reply:
Oia param platform udh ga di pake sii hrsnya. Nanti gw delete deh.